assets/js/index.js

```diff
@@ -0,0 +1,112 @@
+// variables
+const weekDay = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+const startTime = 10;
+const endTime = 20;
+let times = [];
+// let todo = JSON.parse(localStorage.getItem("todoList")) || [];
+
+// get current date and time
+const getCurrentDate = () => {
+  // get current day
+  const day = moment().day();
+  const displayDay = weekDay[day];
+  // get current date
+  const date = moment().toString().split(" ");
+  const displayDate = date[2] + " " + date[1] + " " + date[3];
+  const toDisplay = displayDay + ", " + displayDate;
```

Comment:
Same as the previous comment, please use template literals for anything like this.